src/entity/mod.rs

```diff
@@ -117,3 +117,6 @@ pub use model::*;
 // pub use prelude::*;
 pub use primary_key::*;
 pub use relation::*;
+
+// Imports paginator utility by default making it backward compatible.
+pub use crate::PaginatorTrait;
```

Comment:
I think it should belong to `query::*` instead of `entity::*`

Reply:
Changed on f975223